README.adoc

```diff
@@ -16,31 +16,8 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Development
 
-To run this operator locally you need to have at least one Minishift profile started:
+The dev targets in the toolchain-e2e repository can be used to build and deploy the host and member operators for development: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/make/dev.mk
```

Comment:
I would rather reference the guide https://github.com/codeready-toolchain/toolchain-e2e/blob/master/dev_install.adoc